source/index.html.md

```diff
@@ -355,12 +372,20 @@ ad-hoc.
 For subscriptions, the first in series transaction is a cardholder-initiated
 transaction (CIT) whereas all subsequent transactions are merchant-initiated.
 
-PSPs and merchants may store the cardholder's payment credentials to allow a
-returning cardholder to conveniently pay without entering the payment
-credentials again. (Explicit agreement with Clearhaus is required.)
-To store payment credentials and use these stored payment credentials,
-`credential_on_file[]` must be used. All transactions in such a series are
-cardholder-initiated.
+### Subsequent Credential on file
+
+First in series transactions can also be made using the `applepay` and
+`mobilepayonline` payment methods; however, subsequent merchant-initiated
+transactions must be made using the `card` payment method using the card
+details of the initial recurring authorization.
+
+Subsequent authorizations are made by using
+`credential_on_file[previous]=<ID>`, where `<ID>` is the `id` of the previous
+transaction in the series.
```

Comment:
I've documented it as the ("full") reference, like `/authorizations/:id`, and I have a reason for that. (Hint: "abstract transaction".) 🙂

Reply:
At least I think we need to be a bit more precise than "the previous transaction". Is it the authorization or the capture for the transaction? Also, in my opinion we should mention an age limit, e.g. 13 months.

Reply:
@ct-clearhaus abstract transaction as in?

I agree that we need to be more precise. But we should also be terse.
I'll see what I can do.

Reply:
But I guess `previous` depends on how we accept continued transactions in our API.

Reply:
"Abstract transaction": https://github.com/clearhaus/issues-pci/issues/1071#issuecomment-517264132